ocs_ci/ocs/utils.py

```diff
@@ -770,7 +770,7 @@ def setup_ceph_toolbox(force_setup=False):
             toolbox["spec"]["template"]["spec"]["volumes"].append(ceph_volume)
             rook_toolbox = OCS(**toolbox)
             rook_toolbox.create()
-            return
+
```

Comment:
It was created before and thats why we have return, the patch is only for non-external mode method

Reply:
> It was created before and thats why we have return, the patch is only for non-external mode method

yes, I have added the return statement because if we patch again it with https://github.com/vavuthu/ocs-ci/blob/cc768f60eb1684fa14995bd69ae6539c5b4c8d66/ocs_ci/ocs/utils.py#L777 , it will override the external cluster settings and it ( tool box )  won't even connect to cluster

Reply:
I have observed that if toolbox is created manually, it is removed later from the cluster during reconciliation.
afaik in previous version, running the patch once toolbox was created used to avoid pod removal.
However, as seen by Vijay and in PR job that method is no longer working for us now.

I noted my observations in https://github.com/red-hat-storage/ocs-ci/issues/3941#issuecomment-791307074
Closing this PR as the proposed solution to remove `return` doesn't work.